src/Tasks/Common/Logger.cs

```diff
@@ -62,6 +62,9 @@ public void LogMessage(MessageImportance importance, string format, params strin
         public void LogWarning(string format, params string[] args)
             => Log(CreateMessage(MessageLevel.Warning, format, args));
 
+        public void LogNonSdkWarning(string code, string format, params string[] args)
```

Comment:
Random question, what does non-Sdk prefex mean in this case?

Reply:
we need this if we want the error\warning code doesnt need to be prefixed with string "SDK ERROR"